src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt

```diff
@@ -1,16 +1,30 @@
 package net.maizegenetics.phgv2.cli
 
 import com.github.ajalt.clikt.core.CliktCommand
-import com.github.ajalt.clikt.parameters.arguments.argument
-import com.github.ajalt.clikt.parameters.arguments.default
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.option
-import com.github.ajalt.clikt.parameters.types.int
+import com.github.ajalt.clikt.parameters.options.validate
+
 
 class LoadVcf : CliktCommand() {
 
+    val vcfDir by option(help = "VCF file directory")
```

Comment:
Seems like most these parameters should just be required()?

Reply:
Yes, I will remove the defaults from everything that is required.